clang/test/CodeGen/volatile-1.c

```diff
@@ -1,5 +1,5 @@
 // XFAIL: target=aarch64-pc-windows-msvc
-// RUN: %clang_cc1 -Wno-return-type -Wno-unused-value -emit-llvm %s -w -o - | FileCheck %s
+// RUN: %clang_cc1 -Wno-return-type -Wno-return-mismatch -Wno-unused-value -emit-llvm %s -w -o - | FileCheck %s
```

Comment:
Similarly, why is this change required -- I would have assumed that `-Wno-return-type` was sufficient to silence both. (Same question applies to `return-silent.c`.)

Reply:
same as above. done.